test cases/common/14 configure file/config6.h.in

```diff
@@ -1,19 +1,16 @@
 /* No escape */
 #define MESSAGE1 "@var1@"
 
-/* Single escape means no replace */
-#define MESSAGE2 "\@var1@"
+/* Escaped whole variable */
+#define MESSAGE2 "\\@var1\\@"
```

Comment:
Since the point of this MR was that `\@foo` (without a trailing @) does not work, there should be a test for that exact case.